.appveyor.yml

```diff
@@ -25,42 +25,29 @@ init:
   - git config --global core.autocrlf input
 
 install:
-  - cmd: xb setup
-
-platform: Windows
+  - xb setup
 
 configuration:
   - Release
   - Checked
 
 build_script:
-  - cmd: xb build --config=%CONFIGURATION% --target=src\xenia-app --target=tests\xenia-cpu-ppc-tests --target=src\xenia-vfs-dump
+  - xb build --config=%CONFIGURATION% --target=src\xenia-app --target=tests\xenia-cpu-ppc-tests --target=src\xenia-vfs-dump
 
 after_build:
-  - cmd: |
-      7z a xenia-%appveyor_repo_branch%.zip LICENSE %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia.exe %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia.pdb
-      7z a xenia-vfs-dump-%appveyor_repo_branch%.zip LICENSE %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia-vfs-dump.exe %APPVEYOR_BUILD_FOLDER%\build\bin\%PLATFORM%\%CONFIGURATION%\xenia-vfs-dump.pdb
+  - |-
+      7z a xenia-%APPVEYOR_REPO_BRANCH%.zip LICENSE %APPVEYOR_BUILD_FOLDER%\build\bin\Windows\%CONFIGURATION%\xenia.exe %APPVEYOR_BUILD_FOLDER%\build\bin\Windows\%CONFIGURATION%\xenia.pdb
```

Comment:
There was no reason to change this from `%PLATFORM%`.

Reply:
It was due to `Platform` being removed. Should I put it back?

Reply:
Removed from what, did AppVeyor remove it in a breaking way?

Reply:
I removed it from appveyor.yml. https://github.com/xenia-project/xenia/pull/1495/files#diff-11c909939117928998b102a1fff7d363L30

Reply:
Oh, if removing that also removes the variable then that's fine.

Reply:
It does.